src/pages/calendar/components/Schedule.tsx

```diff
@@ -77,7 +91,7 @@ export function Schedule({ meetingTimes }: ScheduleProps): JSX.Element {
           {/* TODO: verify if we can safely exclude this for most cases */}
           {/* <Th>Schedule Type</Th> */}
           <Th>Location</Th>
-          <Th>Instructors</Th>
+          {professor.length > 0 && <Th>Instructors</Th>}
```

Comment:
I'm confused on the `professor` variable here, since it's still an array will all profs from that course no? why not use m.instructors length directly?

Reply:
I'll clean this up